test/e2e-common.sh

```diff
@@ -93,16 +98,19 @@ function install_knative_serving() {
 # Parameters: $1 - Istio CRD YAML file
 #             $2 - Istio YAML file
 #             $3 - Knative Serving YAML file
+#             $4 - Knative Monitoring YAML file (optional)
 function install_knative_serving_standard() {
   INSTALL_ISTIO_CRD_YAML=$1
   INSTALL_ISTIO_YAML=$2
   INSTALL_RELEASE_YAML=$3
+  INSTALL_MONITORING_YAML=$4
   if [[ -z "${INSTALL_ISTIO_CRD_YAML}" ]]; then
     build_knative_from_source
     INSTALL_ISTIO_CRD_YAML="${ISTIO_CRD_YAML}"
     INSTALL_ISTIO_YAML="${ISTIO_YAML}"
     # TODO(#2122): Install monitoring as well once we have e2e testing for it.
```

Comment:
is this valid still?

Reply:
Removed as https://github.com/knative/serving/issues/2122 is already closed.